packages/astro/test/markdown.test.js

```diff
@@ -17,6 +17,13 @@ describe('Markdown tests', () => {
 			await fixture.build();
 		});
 
+		it('Can load a `.markdown` file', async () => {
+			const html = await fixture.readFile('/page-with-markdown-extension/index.html');
+			const $ = cheerio.load(html);
+			expect($('h1').html()).to.equal('Page with markdown extension');
+			expect($('p').html()).to.equal('Hope this loads fine 🤞');
+		});
```

Comment:
Maybe we need more test cases?

Reply:
I think this is ok to start, i'm not super worried about this change breaking